website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,190 @@
+
+## List
+
+### Positioning
+
+Lists can be positioned to the left, right, or center, as well as above or below the trigger as necessary to fit within the UI. These options are only available when `matchTriggerWidth` is set to `false`.
+
+![Showing the different list positions (left, center, right + top and bottom)](/assets/components/form/super-select/super-select-list-positioning.png =836x*)
+
+### Width
+
+By default, the width of the list matches the trigger width.
+
+![Showing the list's width matching the trigger's](/assets/components/form/super-select/super-select-list-width-matching.png)
+
+#### Auto width
+
+If the list width is set to “auto,” it will match the width of the longest list item. This means it may be either narrower or wider than the trigger width accordingly. A `max-width` can also be defined for the list.
+
+![Showing the list's auto width, resizing to the longest list item](/assets/components/form/super-select/super-select-list-width-auto.png)
+
+## ListItem Types
+
+There are four types of ListItems, two of which are variant-specific.
+
+- `Checkmark` is used in the Single component to indicate a selection has been made.
+- `Checkbox` is used in the Multiple component to indicate that more than one option can be selected.
+- `Title` is used to group ListItems together and help visually organize groups.
+- `Separator` is used to create a visual separation between groups.
+
+![Showing the different list types](/assets/components/form/super-select/super-select-list-item-types.png)
+
+## Groups
+
+ListItems can be grouped visually by adding a title and a separator. The last group in the list doesn’t require a separator.
+
+![Groups of list items represented in both Single and Multiple components](/assets/components/form/super-select/super-select-group-list-items.png =836x*)
+
+## Required and optional
+
+For complex forms, required fields should be indicated. This is the most explicit and transparent method to ensure users don’t have to make assumptions. Read more about best practices for [marking required fields in forms](https://www.nngroup.com/articles/required-fields/).
+
+![Showing required next to the label for the Super Select](/assets/components/form/super-select/super-select-required.png )
+
+For shorter, simpler forms (e.g., login/signup and feedback requests), indicate optional fields instead.
+
+![Showing optional next to the label for the Super Select](/assets/components/form/super-select/super-select-optional.png )
+
+## Error validation
+
+For error validation recommendations, refer to the [Form patterns](/patterns/form-patterns?tab=validation) documentation.
+
+## Content
+
+For general content recommendations, refer to the [Primitives](/components/form/primitives) documentation.
+
+### Options list
+
+- We recommend keeping option content clear and concise; avoid full sentences.
+- Avoid repeating the same word at the beginning of a set of options.
```

Comment:
For this one, I'm not clear what it means. Does it mean, don't use the same word to begin each option?

Reply:
Yes. Maybe we should just say:
```suggestion
- Avoid repeating the same word at the beginning of list options.
```

Reply:
Thanks